src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -77,6 +84,25 @@
 
     private final Random random = new Random(System.currentTimeMillis());
 
+    private static final RepoLabels labelsYAML;
+    static {
+        try {
+            URL url = new URL("https://raw.githubusercontent.com/jenkinsci/.github/4f59a6219705df14067688df4f2a503e143cfa92/labels.yaml");
```

Comment:
Point to master branch?

Reply:
master doesn't have the file so it would 404, but yea the plan would be master